test/Parse/line-directive.swift

```diff
@@ -24,6 +24,13 @@ x x // expected-error{{consecutive statements}} {{2-2=;}}
 
 // rdar://19582475
 public struct S { // expected-note{{in declaration of 'S'}}
+// expected-error@+8{{expected 'func' keyword in function declaration}}
+// expected-error@+7{{operator '/' declared in type 'S' must be 'static'}}
+// expected-error@+6{{expected '(' in argument list of function declaration}}
+// expected-error@+5{{operators must have one or two arguments}}
+// expected-error@+4{{member operator '/()' must have at least one argument of type 'S'}}
+// expected-error@+3{{expected '{' in body of function declaration}}
+// expected-error@+2{{consecutive declarations on a line must be separated by ';}}
```

Comment:
Hrm, this is unfortunate. I guess it's hard to catch the generic case without it though…